### ISSUE TITLE
[11.0] web_widget_x2many_2d_matrix: can I use m2o field inside the matrix?

### ISSUE THREAD

**Issue by jaronemo** [OCA/web#1067]
Dears:

can I use m2o field inside the matrix?

**Comment by simahawk** [comment]
can you rephrase? Do you mean: "can I use m2o field inside the matrix"?

**Comment by pedrobaeza** [comment]
@simahawk he rephrased, but without advertising it. Can you answer?

**Comment by simahawk** [comment]
@jaronemo starting from v11 the widget loads all the inner fields using their own widget.
So - potentially - it should work. I say "potentially" because I've never had this need so far.
Are you facing any particular issue?

**Comment by GSLabIt** [comment]
@simahawk, probably this is better thread, do u know if it is possible to show fields not included in matrix (before/after it)? Thx

**Comment by simahawk** [comment]
@GSLabIt I think you've got the answer on  #1264  :)

**Comment by simahawk** [comment]
@jaronemo closing as you got the answer. Feel free to re-open if needed.
